index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Collaboration Henry & Imran</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    
+      <h1 id="title" style="text-align: center; text-transform: uppercase;">Microverse Survey Form</h1>
+      <p style="text-align: center; text-transform: uppercase; " id="description">This Survey form is intented to collect data of all prospective students of Microverse</p>
+   
+      <div style="margin-top: 4%;"></div>
+
+    <form id="survey-form" action="">
+      <p style="font-weight: bold; text-transform: uppercase;">Enter Your Personal Details</p>
+      <label for="name" id="name-label">Name</label> <br>
+    <input type="text" name="" id="name" placeholder="Please Enter Your name" required>
+    <br>
+
+      <label for="name" id="email-label">Email</label>
+      <br>
+      <input type="email" name="" id="email" placeholder="Please Enter Your Email" required>
+      <br>
+
+      <label for="name" id="number-label">Number</label><br>
+      <input type="number" max="50" min="1" name="" id="number" placeholder="Please Enter Your Number" > <br>
+      <br>
+      <p style="font-weight: bold; text-transform: uppercase;">Which option best describes your current role?</p>
```

Comment:
kindly avoid using inline CSS and consider using external CSS. it allow you to edit the look of a entire page by making changes to just one file, and it is best practice.